TombEngine/Game/gui.cpp

```diff
@@ -3033,7 +3033,7 @@ namespace TEN::Gui
 						{
 							if (numItems == -1)
 							{
-								sprintf(textBuffer, "Unlimited %s", g_GameFlow->GetString(invObject.ObjectName));
```

Comment:
Why do you use `strcat` and `strdup` here? Just make `%$` mask part of the lua string. User may want to appear ammo name prior to "unlimited", or it may be language-specific.